File: API/Helpers/UserParameters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Helpers
{
    public class UserParameters : PaginationParameters
    {
        public string CurrentUsername { get; set; }
        public int BrandId { get; set; }
        public int ModelId { get; set; }
        public int MinEnginePower { get; set; } = 0;
        public int MaxEnginePower { get; set; } = 1500;
        public string OrderBy { get; set; } = "CreateDateDesc";
    }
}

File: client/src/app/models/userParameters.ts
import { User } from "./user";

export class UserParameters {
    brandId = 0;
    modelId = 0;
    minEnginePower = 0;
    maxEnginePower = 1500;
    pageNumber = 1;
    pageSize = 20;
    orderBy = "CreateDateDesc";
}